node/tests/actors/rad_manager.rs

```diff
@@ -22,6 +22,11 @@ fn from_config_test_success_helper(
         .cloned()
         .map(|option| option.map(|uri| uri.to_string()))
         .collect::<Vec<_>>();
+    // This is dropping RadManager explicitly so as to get rid of the panic of doing so.
+    std::panic::catch_unwind(|| {
+        std::mem::drop(manager);
+    })
+    .ok();
```

Comment:
Ideally this shouldn't be needed, I believe there is a bug in the `stop_system_if_panicking` function, which panics if the actor is being dropped in a panicking thread while no actix system is running. When testing other actors they always run inside an actix system so that's not a problem, but the RadManager is more standalone. So I believe that running inside an actix system should be optional, to avoid doing this workaround in other code that doesn't use actix. I'll see if it can be fixed.

Reply:
Done, #2276

Reply:
Thank you!

Reply:
So this hack can be removed now?